src/other-reprs.md

```diff
@@ -143,7 +143,7 @@ This is a modifier on `repr(C)` and `repr(rust)`. It is incompatible with
 
 
 
-[reference]: https://github.com/rust-rfcs/unsafe-code-guidelines/tree/master/reference/src/representation.md
+[reference]: https://rust-lang.github.io/unsafe-code-guidelines/layout.html
```

Comment:
I'm confused why this is referring to the UCG rather than the actual reference...
Would be good to change the text itself to say "unsafe code guidelines" and then explicitly note that it's also not normative.

Reply:
Indeed. Updated the sentence also.